homeassistant/components/vacuum/dyson.py

```diff
@@ -7,14 +7,14 @@
 import asyncio
 import logging
 
+from homeassistant.util.icon import icon_for_battery_level
```

Comment:
Ok, this time it should be fine. Next time I'll try to do it in the first commit :)